perun-core/src/main/java/cz/metacentrum/perun/core/entry/UsersManagerEntry.java

```diff
@@ -515,6 +515,20 @@ public UserExtSource getUserExtSourceById(PerunSession sess, int id) throws User
 		return getUsersManagerBl().getUserExtSourceById(sess, id);
 	}
 
+	@Override
+	public List<UserExtSource> getUserExtSourcesByIds(PerunSession sess, List<Integer> ids) throws PrivilegeException {
+		Utils.checkPerunSession(sess);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "getUserExtSourcesByIds_List<Integer>_policy")) {
+			throw new PrivilegeException(sess, "getUserExtSourcesByIds");
+		}
+		List<UserExtSource> userExtSources = getUsersManagerBl().getUserExtSourcesByIds(sess, ids);
+		userExtSources.removeIf(ues -> !AuthzResolver.authorizedInternal(sess, "filter-getUserExtSourcesByIds_List<Integer>_policy", ues));
+
+		return userExtSources;
```

Comment:
Due to changes in perun-roles.yml you should modify this method.

Reply:
Done